pact.cabal

```diff
@@ -1,20 +1,20 @@
+cabal-version:       2.2
```

Comment:
We discussed this and while I'm usually very conservative about these kinds of bumps, this one will probably make sense eventually so it seems like it's probably worth doing now before launch.